modules/nf-core/megan/rma2info/tests/main.nf.test

```diff
@@ -0,0 +1,61 @@
+nextflow_process {
+
+    name "Test Process MEGAN_RMA2INFO"
+    script "../main.nf"
+    process "MEGAN_RMA2INFO"
+
+    tag "modules"
+    tag "modules_nfcore"
+    tag "megan"
+    tag "megan/rma2info"
+
+    test("malt - rma") {
+
+        when {
+            process {
+                """
+                input[0] = [
+                    [ id:'test', single_end:false ], // meta map
+                    file(params.modules_testdata_base_path + 'delete_me/malt/test.rma6', checkIfExists: true)
+                ]
+
+                input[1] = 'true'
+                """
+            }
+        }
+
+        then {
+            assertAll(
+                { assert process.success },
+                { assert snapshot ( process.out.versions ).match() },
+                { assert file (process.out.txt.get(0).get(1)).exists() }
```

Comment:
any chance you can capture more of the content than just that the file exists, eg some lines or is very unstable?

Reply:
It's a semi-binary file IIRC...

Reply:
Snapshot THAT

![image](https://github.com/user-attachments/assets/ae9f1168-f9bb-4b94-aa5f-dc903838720d)

Reply:
![arnold-schwarzenegger-i-give-up-gif-by-jerology](https://github.com/user-attachments/assets/bb3086da-e866-4a4b-b1cb-fa47f41f63da)